tower-grpc-build/src/lib.rs

```diff
@@ -226,8 +226,9 @@ fn to_upper_camel(s: &str) -> String {
 }
 
 /// This function takes an array of comments and properly formats them.
```

Comment:
Tiny nit: I think this comment ought to be updated now that it takes a `&prost::Comments`.

Also, I _might_ rephrase  "properly formats them" as "formats them as a `String` to be used as a RustDoc comment", or something?